### DISCUSSION TITLE
Welcome to odata.net Discussions!

### DISCUSSION THREAD

**Discussion by xuzhg** [OData/odata.net#2403] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wortho** [comment]
Why has https://www.nuget.org/packages/Microsoft.OData.Core/7.11.0 been unlisted? 
"The owner has unlisted this package. This could mean that the package is deprecated, has security vulnerabilities or shouldn't be used anymore."
Do we need to downgrade to 7.10.0 ?
